packages/eslint-plugin-pf-codemods/lib/helpers.js

```diff
@@ -74,7 +74,10 @@ function moveSpecifiers(
         return (
           node?.type === nodeType &&
           [modifiedPackage, toPackage].includes(node?.source?.value) &&
-          specifierReference?.parent?.importKind === node?.importKind
+          (
+            specifierReference?.parent?.importKind === node?.importKind &&
+            specifierReference?.parent?.exportKind === node?.exportKind
```

Comment:
i figured i'd just do && here rather than checking if we are dealing with import or export and then checking that one... this works because if we are dealing with an import... exportKind will be undefined in both places and vice versa